### PR TITLE
Fix rebalance issues

### PR DESCRIPTION
**Pull request by savonarola**
Fixes:

* Error formatting issues in rebalance API
[EMQX-9894](https://emqx.atlassian.net/browse/EMQX-9894)
[EMQX-9895](https://emqx.atlassian.net/browse/EMQX-9895)
[EMQX-9941](https://emqx.atlassian.net/browse/EMQX-9941)
* `wait_health_check` option
[EMQX-9950](https://emqx.atlassian.net/browse/EMQX-9950)

We fix error message formatting in rebalance API: previously they could be displayed as iolist dumps (internal Erlang structure).
We add `wait_health_check` option to node evacuation CLI and API. This is a time interval when the node reports "unhealthy status" without beginning actual evacuation. We need this to allow a Load Balancer (if any) to remove the evacuated node from balancing and not forward (re)connecting clients to the evacuated node.

https://github.com/emqx/emqx-docs/pull/1925

### <samp>🤖 Generated by Copilot at d32ca8b</samp>

This pull request enhances the node rebalance and evacuation features of emqx by adding a `wait_health_check` option to the API and CLI, improving the error message formatting, and adjusting the timeouts and options for better performance and reliability. It also updates the tests and the documentation to reflect the changes.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [x] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [x] Schema changes are backward compatible

### CODE REVIEW

File: apps/emqx_node_rebalance/src/emqx_node_rebalance.erl [hunk 1]
```diff
@@ -438,7 +438,7 @@ is_node_available() ->
     node().
 
 all_nodes() ->
-    mria_mnesia:running_nodes().
+    mria:running_nodes().
```
Comment (thalesmg) [inline apps/emqx_node_rebalance/src/emqx_node_rebalance.erl:441]: Nit just for consistency:

```suggestion
    emqx:running_nodes().
```

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_node_rebalance/src/emqx_node_rebalance_evacuation_persist.erl
[the remark this answers is not in the document]
Comment (savonarola) [reply apps/emqx_node_rebalance/src/emqx_node_rebalance_evacuation_persist.erl:1]: No: getting options from a file generally means that a node was restarted, so it cannot have received connections gracefully. So we omit the graceful procedure.

**Comment by yanzhiemq** [comment]
_

**Comment by yanzhiemq** [comment]
_

**Pull request merged**